component-server-parent/component-server/src/test/java/org/talend/sdk/component/server/front/ComponentResourceImplTest.java

```diff
@@ -179,6 +178,23 @@ void migrate() {
         assertEquals("true", migrated.get("migrated"));
     }
 
+    @Test
+    void no_migrate() {
```

Comment:
Please avoid underscore in method's name.
```suggestion
    void protectHigherMigration() {
```
or 
```suggestion
    void noMigration() {
```